images.md

```diff
@@ -97,15 +90,13 @@ The following is a list of container images maintained in this organization. Mos
 | [6-toolchain](https://github.com/sclorg/devtoolset-container/tree/master/6-toolchain) | [rhscl/devtoolset-6-toolchain-rhel7](https://access.redhat.com/containers/#/registry.access.redhat.com/rhscl/devtoolset-6-toolchain-rhel7) | [centos/devtoolset-6-toolchain-centos7](https://hub.docker.com/r/centos/devtoolset-6-toolchain-centos7/) |
 | [6-perftools](https://github.com/sclorg/devtoolset-container/tree/master/6-perftools) | [rhscl/devtoolset-6-perftools-rhel7](https://access.redhat.com/containers/#/registry.access.redhat.com/rhscl/devtoolset-6-perftools-rhel7) | [centos/devtoolset-6-perftools-centos7](https://hub.docker.com/r/centos/devtoolset-6-perftools-centos7/) |
 
-
 ## Redis
 
 | **version**  | **Red Hat Catalog** | **Docker Hub** |
 | ------------ | ------------------- | -------------- |
 | [3.2](https://github.com/sclorg/redis-container/tree/master/3.2) | [rhscl/redis-32-rhel7](https://access.redhat.com/containers/#/registry.access.redhat.com/rhscl/redis-32-rhel7) | [centos/redis-32-centos7](https://hub.docker.com/r/centos/redis-32-centos7/) |
 | [5](https://github.com/sclorg/redis-container/tree/master/5) | [rhscl/redis-5-rhel7](https://access.redhat.com/containers/#/registry.access.redhat.com/rhscl/redis-5-rhel7) |  |
 
-
 ## MongoDB
```

Comment:
And also for MongoDB.
We support following container repositories:
httpd-container
nginx-container
s2i-php-container
varnish-container
mariadb-container
postgresql-container
s2i-nodejs-container
s2i-python-container
mysql-container
redis-container
s2i-perl-container
s2i-ruby-container